core/src/main/java/org/n52/iceland/request/AbstractServiceRequest.java

```diff
@@ -175,6 +180,18 @@ public Locale getRequestedLocale() {
         return LocaleHelper.fromString(getRequestedLanguage());
     }
 
+    public String getOriginalRequest() {
```

Comment:
Removing it would instantly require a 2.0 release. But for me, the main reason is that it's really used a lot. It would be quite some hassle to change this and will require a lot of changes in the SOS...